test/src/module/integration/allTest.c

```diff
@@ -249,6 +249,9 @@ testRun(void)
             // Stop the cluster
             HRN_HOST_PG_STOP(pg1);
 
+            hrnHostConfigUpdateP(
+                .archiveAsync = BOOL_TRUE_VAR, .spoolPath = VARSTR(strNewFmt("%s/1", strZ(hrnHostSpoolPath(pg1)))));
```

Comment:
We probably shouldn't repeat the same thing 7 times. Maybe write a function or a macro for this?